src/components/Todo/TodoFooter.tsx

```diff
@@ -0,0 +1,62 @@
+import cn from 'classnames';
+import { FilterStatus } from '../../types/Filter';
+import { toCapitalCase } from '../../utils/toCapitalCase';
+import { useTodoContext } from '../../store/todoContext';
+
+export const TodoFooter = () => {
+  const {
+    // prettier-ignore
+    changeFilterStatus,
+    filterStatus,
+    todoLength,
+    clearCompletedTodos,
+    completedTodos,
+  } = useTodoContext();
+
+  const filterStatusList = Object.keys(FilterStatus) as Array<
+  keyof typeof FilterStatus
+  >;
+
+  return (
+    <footer
+      className="todoapp__footer"
+      data-cy="Footer"
+    >
+      <span
+        className="todo-count"
+        data-cy="todosCounter"
+      >
+        {todoLength > 0 && `${todoLength} items left`}
+      </span>
+
+      <nav
+        className="filter"
+        data-cy="Filter"
+      >
+        {filterStatusList.map(status => (
```

Comment:
I think here you can just create each link instead of map, it's only 3 options. This way you don't need extra variables, functions, types etc.